tests/integration/API/MP/CustomFieldsTest.php

```diff
@@ -80,4 +81,30 @@ public function testItReturnsCustomFields() {
       ],
     ]);
   }
+
+  public function testItCreateNewCustomField() {
+    $response = $this->api->addSubscriberField([
+      'name' => 'text custom field',
+      'type' => 'text',
+      'params' => [
+        'required' => '1',
+        'label' => 'text custom field',
+        'date_type' => 'year_month_day',
+      ],
+    ]);
+    expect($response)->array();
+    expect($response)->hasKey('name');
```

Comment:
What are your thoughts on making the assertions on this test more specific and check that the full response matches what was expected instead of just checking if the response is an array and has a 'name' key?

Reply:
Well, my thinking was that if the array is not empty the method succeeded and so we don't need to assert all the fields like in the `get` method.

Reply:
I don't have a strong opinion on this one. I would've written an assertion to check if the method is returning exactly what we expect it to return, but if you prefer to leave it the way it is, it is fine as well.

My concern is that this test will pass even if in the future we inadvertently introduce a change that, for example, returns the key 'typo' instead of 'type' (ok, not the best example).